bin/build-demo.js

```diff
@@ -9,33 +9,24 @@ if (!fs.existsSync('build/loader/loader-debug.js')) {
 
 copy('build/loader/loader-debug.js', 'build/demo/loader-debug.js');
 copy('src/demo/index.html', 'build/demo/index.html');
-copy('src/demo/modules2', 'build/demo/modules2');
-copy('src/demo/modules3', 'build/demo/modules3');
-copy('src/demo/packages', 'build/demo/packages');
+copy('src/demo/config.js', 'build/demo/config.js');
+
+fs.mkdirsSync(`build/demo/resolutions`);
+globby.sync('src/demo/resolutions/*').forEach(file => {
+	const filePath = file.substring('src/demo/resolutions/'.length);
+
+	copy(
+		`src/demo/resolutions/${filePath}`,
+		`build/demo/resolutions/${filePath}`
+	);
+});
 
 globby.sync('src/demo/modules/**/*.js').forEach(file => {
+	console.log(file);
 	const filePath = file.substring('src/demo/modules/'.length);
 	const dirname = path.dirname(filePath);
 
 	fs.mkdirsSync(`build/demo/modules/${dirname}`);
-	run(
-		'babel',
-		`src/demo/modules/${filePath}`,
-		'--out-file',
-		`build/demo/modules/${filePath}`
-	);
-});
 
-fs.mkdirsSync(`build/demo/config`);
-run(
-	'liferay-cfgen',
-	'-b',
-	'src/demo/config/config-base.js',
-	'-m',
-	'src/demo/modules/bower.json',
-	'-o',
-	'build/demo/config/config.js',
-	'-r',
-	'build/demo/modules',
-	'build/demo/modules'
-);
+	copy(`src/demo/modules/${filePath}`, `build/demo/modules/${filePath}`);
```

Comment:
I'm changing it to `path.*`. Using the '/' is generally safe in Windows for composing paths because `/` paths are supported in the core of the OS. But I agree that it leads to subtle errors when parsing or splitting paths, so it's better to stick to `\` just in case. 👍

Reply:
Regarding the namespacing, I usually prefer to be verbose because `join` can be anything, but `path.join` shows what it is without any doubt. 

My rule of thumb is: unless there's no possible confusion I prefer to namespace.